hadoop-mapreduce/src/main/java/org/apache/accumulo/hadoop/mapreduce/partition/RangePartitioner.java

```diff
@@ -14,7 +14,7 @@
  * See the License for the specific language governing permissions and
  * limitations under the License.
  */
-package org.apache.accumulo.hadoop.mapreduce.lib.partition;
+package org.apache.accumulo.hadoop.mapreduce.partition;
```

Comment:
Is this the best name for this? Should the package name end with "partition", or should it be "partitioners" (set nomenclature), or "partitioning" (category nomenclature).